README.md

```diff
@@ -51,15 +55,27 @@ Set up ENV vars in every environment:
   * `example.org`
   * etc
 
+### Prepare your controllers
 
-Mix `TestTrack::Controller` into any controllers needing access to TestTrack:
+Mix `TestTrack::Controller` into any controllers needing access to TestTrack and configure it with the name of your `:current_user` method.
 
 ```ruby
 class MyController < ApplicationController
   include TestTrack::Controller
+
+  self.test_track_identity = :current_user
 end
 ```
 
+### Prepare your identity models
+
+You'll need to configure your `User` model as a [TestTrack Identity](#varying-app-behavior-from-within-a-model)
+
+If your app doesn't support authentication, just define a method that
+always returns `nil`, e.g. `def current_user; end`.
```

Comment:
This just became non-optional because it's the primary mechanism for visitor lookup in an authenticated state, so I added it to the installation docs.

Reply:
Where does this method need to be defined?

Reply:
We should mention that this goes in the controller.